fiftyone/utils/hrm2.py

```
@@ -0,0 +1,2434 @@
+
+    # Clear dataset to rebuild with groups
+    logger.info("Creating grouped dataset structure...")
+    dataset.clear()
+
+    # Set up the dataset for groups
+    dataset.add_group_field("group", default=image_slice_name)
+
+    # Create all new samples with groups
+    all_new_samples = []
+
+    for orig_data, pred in zip(original_data, predictions):
```

Ruff (0.14.5)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)